aiger_analysis/cadet.py

```diff
@@ -57,6 +59,7 @@ def simplify_quantifier_prefix(quantifiers):
     seen_variables = set()
     simplified_q = []
     for q, variables in quantifiers:
+        variables = list(map(str, variables))
```

Comment:
Why not just `set(map(str, variables))`.

Reply:
Played around with it. In the end it should better be a list and preserve the order of variables, I feel.